src/lib/components/SiteFooter/index.jsx

```diff
@@ -315,10 +324,22 @@ class SiteFooter extends React.Component {
   }
 }
 
+const enterpriseQueryParamsShape = {
+  utm_campaign: PropTypes.string,
+  utm_source: PropTypes.string,
+  utm_medium: PropTypes.string,
+};
+
+const enterpriseLinkShape = {
+  url: PropTypes.string,
+  queryParams: PropTypes.shape(enterpriseQueryParamsShape),
+};
+
 SiteFooter.propTypes = {
   siteName: PropTypes.string,
   siteLogo: PropTypes.node,
   marketingSiteBaseUrl: PropTypes.string,
+  enterpriseMarketingLink: PropTypes.shape(enterpriseLinkShape),
```

Comment:
You can just include the shape for both the query params and the link directly in the `propTypes` object, as opposed to creating separate variables for them. Related, I'm not too sure we need to specifically call out the query params shape.

I think just the following would suffice here:
```jsx
SiteFooter.propTypes = {
  enterpriseMarketingLink: PropTypes.shape({
    url: PropTypes.string.isRequired,
    queryParams: PropTypes.shape({}),
  }),
};
```

Reply:
👍